Dockerfile

```diff
@@ -1,110 +1,85 @@
     for name in SSSP_efficiency_pseudos SSSP_precision_pseudos; do \
-       wget ${base_url}/${name}.aiida;                             \
+       wget ${base_url}/${name}.tar.gz;                            \
+       tar -zxvf ${name}.tar.gz;                                   \
     done;                                                          \
     chown -R root:root /opt/pseudos/;                              \
     chmod -R +r /opt/pseudos/
 
-## install rclone
-#WORKDIR /opt/rclone
-#RUN wget https://downloads.rclone.org/rclone-v1.38-linux-amd64.zip;  \
-#    unzip rclone-v1.38-linux-amd64.zip;                              \
-#    ln -s rclone-v1.38-linux-amd64/rclone .
-
-# install PyPI packages for Python 3
+# install packages that are not in the aiidalab meta package
+# 'fastentrypoints' is to fix problems with aiida-quantumespresso plugin installation
 RUN pip3 install --upgrade         \
-    'tornado==5.0.2'               \
+    'fastentrypoints'              \
+    'tornado==5.1.1'               \
     'jupyterhub==0.9.4'            \
-    'notebook==5.5.0'              \
-    'nbserverproxy==0.8.3'         \
+    'notebook==5.7.4'              \
+    'nbserverproxy==0.8.8'         \
+    'jupyterlab==0.35.4'           \
     'appmode-aiidalab==0.5.0.1'
 
 # enable nbserverproxy extension
 RUN jupyter serverextension enable --sys-prefix --py nbserverproxy
 
-# install PyPI packages for Python 2.
 # This already enables jupyter notebook and server extensions
-RUN pip install aiidalab==v19.05.2
+RUN pip3 install aiidalab==v19.11.0a2
+
+# activate ipython kernel
+RUN python3 -m ipykernel install
 
+# TODO: delete, when https://github.com/aiidalab/aiidalab-widgets-base/issues/31 is fixed
 # the fileupload extension also needs to be "installed"
 RUN jupyter nbextension install --sys-prefix --py fileupload
 
-## Get latest bugfixes from aiida-core
-#RUN pip2 install --no-dependencies git+https://github.com/ltalirz/aiida_core@v0.12.1_expire_on_commit_false
-
-# Install editable aiida version
-#WORKDIR /opt/aiida-core
-#RUN git clone https://github.com/aiidateam/aiida_core.git && \
-#    cd aiida_core && \
-#     git checkout release_v0.11.2 && \
-#     pip install --no-deps . && \
-#    cd ..
-
-# activate ipython kernels
-RUN python2 -m ipykernel install
-RUN python3 -m ipykernel install
+# Enable nbserverproxy extension.
+RUN jupyter serverextension enable --sys-prefix --py nbserverproxy
+# Enables better integration with jupyterhub.
+# https://jupyterlab.readthedocs.io/en/stable/user/jupyterhub.html#further-integration
+RUN jupyter labextension install @jupyterlab/hub-extension
+
+# Install jupyterlab theme.
+WORKDIR /opt/jupyterlab-theme
+RUN git clone https://github.com/aiidalab/jupyterlab-theme && \
+    cd jupyterlab-theme && \
+     npm install && \
+     npm run build && \
+     npm run build:webpack && \
+     npm pack ./ && \ 
+     jupyter labextension install *.tgz && \
+    cd ..
 
 # install MolPad
```

Comment:
do we use this in any way?

Reply:
I don't think so. Should we remove it here, or maybe in a separate PR?

Reply:
fine to remove in another PR